tests/providers/azure/services/defender/defender_auto_provisioning_log_analytics_agent_vms_on/defender_auto_provisioning_log_analytics_agent_vms_on_test.py

```diff
@@ -0,0 +1,90 @@
+from unittest import mock
+from uuid import uuid4
+
+from prowler.providers.azure.services.defender.defender_service import AutoProvisioningSetting
+
+AZURE_SUSCRIPTION = str(uuid4())
```

Comment:
You can use this value from the `tests/prowler/providers/azure/azure_fixtures.py`. Pedro included that in the previous PR.